addons/beehave/nodes/composites/selector.gd

```diff
@@ -27,7 +27,9 @@ func tick(actor: Node, blackboard: Blackboard) -> int:
 			FAILURE:
 				c.after_run(actor, blackboard)
 			RUNNING:
-				running_child = c
+				if c != running_child:
```

Comment:
This looks wrong to me, as we assign `running_child` to `c` prior, so it would never be different. We might need to first do this check before assigning `c` to `running_child`